docs/content/en/docs/concepts/architecture/deployment-flow.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Flow of deployment
+linktitle: Flow of deployment
+description: Understand the execution flow of a deployment
+weight: 35
+---
+
+The Keptn Lifecycle Toolkit (KLT) deploys a
+[Kubernetes Workload](https://kubernetes.io/docs/concepts/workloads/)
+by passing through a well-defined execution flow.
+
+The execution flow goes through six main phases:
+
+* Pre-deployments-tasks
+* Pre-deployment-evaluation
+* Deployment
+* Post-deployment-tasks
+* Post-deployment-evaluation
+* Completed
+
+Within each phase,
```

Comment:
Actually, the tasks within each phase are all executed in parallel

Reply:
done